feature-libs/cart/main/root/cart-root.module.ts

```diff
@@ -24,6 +27,20 @@ export function defaultCartComponentsConfig() {
           'SaveForLaterComponent',
         ],
       },
+      [MINI_CART]: {
+        cmsComponents: ['MiniCartComponent'],
+        module: () =>
+          import('@spartacus/cart/main/components/cart/mini-cart').then(
+            (m) => m.MiniCartModule
+          ),
+      },
+      [ADD_TO_CART]: {
+        cmsComponents: ['ProductAddToCartComponent'],
+        module: () =>
+          import('@spartacus/cart/main/components/cart/add-to-cart').then(
+            (m) => m.AddToCartModule
+          ),
+      },
```

Comment:
Does this work?

I was under the impression that dynamic imports only woorked from the shell app.  But perhaps this works since rot is eager loaded.  Lets discuss.

Reply:
it works

Reply:
In a way, I was hoping this would work.  If this needs to move to the shell app, we have to update the installation schematics to support it.